tests/TestCase.php

```diff
@@ -24,7 +23,25 @@
  */
 abstract class TestCase extends BaseTestCase
 {
-    use ExpectDeprecationTrait;
+    /** @var null|callable */
+    private $previouslyDefinedErrorHandler;
+
+    /** @var array<string> */
```

Comment:
```suggestion
    /** @var list<string> */
```

if indexes are 0..n and not `1, 5, 10`, I want to use `list` for whole project